packages/kernel-6.12/config-bottlerocket

```diff
@@ -48,6 +48,40 @@ CONFIG_FB=y
 CONFIG_FB_EFI=y
 CONFIG_FRAMEBUFFER_CONSOLE_DEFERRED_TAKEOVER=y
 
+# Config options for kernel hardening.
+
+# Do not allow writes to a block device if a filesystem
+# is mounted on it.
+# CONFIG_BLK_DEV_WRITE_MOUNTED is not set
+
+# Bottlerocket does not hibernate
+# CONFIG_HIBERNATION is not set
+# Do not permit processes to push characters into
+# their controlling tty device
+# CONFIG_LEGACY_TIOCSTI is not set
+
+# Bottlerocket does not livepatch
+# CONFIG_LIVEPATCH is not set
+
+# The "magic sysrq" key is enabled. Set
```

Comment:
nit: you can use a much longer line length for all of these comments, 80 should be fine.